qiskit_jku_provider/qasm_simulator_jku.py

```diff
@@ -23,13 +23,16 @@
 from collections import OrderedDict, Counter
 import numpy as np
 
-from qiskit.backends import BaseBackend
+#from qiskit.backends import BaseBackend
```

Comment:
Can you make a round of removing commented-out lines similar to this one, for readability? Usually we can just refer to git history to see what lines have changed if needed!

Reply:
You are correct, of course. Done.